pkg/ucp/integrationtests/resourcegroups/resourcegroups_test.go

```diff
@@ -40,13 +39,13 @@ const (
 	resourceGroupInvalidResponseFixture = "testdata/resourcegroup_invalid_v20231001preview_responsebody.json"
 )
 
-func createRadiusPlane(server *testserver.TestServer) {
+func createRadiusPlane(server *testhost.TestHost) {
```

Comment:
See other comments about test helpers. I think these could be consolidated, but I won't be doing that work in this PR.